### ISSUE TITLE
[Geodata] Mourkain Temple wall

### ISSUE THREAD

**Issue by SODIX** [WarEmu/WarBugs#9568]
In some cases(I just don't get how ppl do this) that wall goes into full transparent mode. And so I seen wallhack bug there.
![war 2017-04-29 21-05-28](https://cloud.githubusercontent.com/assets/22780561/25711690/3d414f8e-30f8-11e7-866f-b9cda0e8d39e.png)

**Comment by Luth-RoR** [comment]
Do you mean that the wall disappears randomly?

**Comment by SODIX** [comment]
No, players just walk through it. Like there is no a wall at all.

**Comment by SODIX** [comment]
Any news. Would it be fixed by any chance?
